bucket/hello-http.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": "1.4.1",
+    "description": "A cross-platform HTTP client desktop application for testing HTTP and REST APIs, WebSocket, GraphQL (including subscriptions) and gRPC endpoints.",
+    "homepage": "https://sunny-chung.github.io/hello-http/",
+    "url": "https://github.com/sunny-chung/hello-http/",
```

Comment:
```suggestion
```

This appearance should be removed (since considered as link to the files to download).

Reply:
but why JSON schema still contains this field as required?
I added url because IDEA suggest me, it makred  are required
https://github.com/ScoopInstaller/Scoop/blob/master/schema.json#L649-L651